.gitignore

```diff
@@ -1 +0,0 @@
-.DS_Store
```

Comment:
@Stephen-Roach You definitely don't want to delete this file. I don't think we've talked about it yet. Remind me in class tomorrow if I don't mention it.